.github/workflows/sanitizer-build-and-test.yaml

```
@@ -0,0 +1,157 @@
+name: Sanitizer test
+on:
+  schedule:
+    # run daily 20:00 on master branch
+    - cron: '0 20 * * *'
+  push:
+    branches:
+      - prerelease_test
+jobs:
+  sanitizer:
+    name: PG${{ matrix.pg }} ${{ matrix.name }} ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        name: ["Sanitizer"]
+        pg: ["12.7", "13.3"]
+        os: ["ubuntu-20.04"]
+        build_type: ["Debug"]
+
+        # llvm-symbolizer crashes so we don't use clang until we resolve that
+        extra_packages: ["clang-10 llvm-10 llvm-10-dev llvm-10-tools"]
+        llvm_config: [llvm-config-10"]
+        clang: ["clang-10"]
+
+        # gcc-10 is the minimum version that can parse the suppression files barely adequately
+        cc: ["gcc-10"]
+        cxx: ["g++-10"]
+
+    env:
+      PG_SRC_DIR: pgbuild
+      PG_INSTALL_DIR: postgresql
+      MAKE_JOBS: 6
+      CLANG: ${{ matrix.clang }}
+      CC: ${{ matrix.cc }}
+      CXX: ${{ matrix.cxx }}
```

Do you need this at all? We're not building any C++ code, right?
```suggestion
      CXX: g++-10
```

---

postgresql has some C++